inference_rules.adoc

```diff
@@ -821,6 +821,30 @@ The DLRMv2 MLPerf inference code has an option to aggregate multiple consecutive
 2. drawn uniformly from interval [`--samples-to-aggregate-min`, `--samples-to-aggregate-max`]	
 3. drawn from a custom distribution, with its quantile (inverse of CDP) specified in `--samples-to-aggregate-quantile-file=./tools/dist_quantile.txt`.
 
+=== LLM Benchmarks
+
+
+LLM benchmarks contain an auto-regressive generation loop. This loop’s execution is controlled by a decoding algorithm. The following rules discuss the decoding algorithm in the light of MLPerf Inference rules for the Closed submissions:
+
```

Comment:
In the whole PR:
does "should" mean "should" from [https://www.ietf.org/rfc/rfc2119.txt](https://www.ietf.org/rfc/rfc2119.txt) or "must"?